src/main/kotlin/org/rust/lang/refactoring/RustNameSuggestions.kt

```diff
@@ -28,6 +26,7 @@ fun PsiElement.suggestNames(): LinkedHashSet<String> {
     val foundNames = when {
         this.isArgument() -> rustNameUtil(nameForArgument())
         this is RustCallExprElement -> nameForCall(this).flatMap(::rustNameUtil)
+        isStructField() -> rustNameUtil((this.parent as? RustStructExprFieldElement)?.identifier?.text ?: "")
```

Comment:
I would have just saved `parent` in the local variable and then inlined the functions:

```
val parent = parent //
val foundNames = when {
    if parent is RustStructFieldElement -> .... 
}
```